### DISCUSSION TITLE
Datepicker after upgrade

### DISCUSSION THREAD

**Discussion by pyinstallerbaggins** [marimo-team/marimo#2223] [Q&A]
After upgrading to marimo==0.8.9

My datepicker now lost the extra capabilities to select month, year like in the documentation below
![image](https://github.com/user-attachments/assets/bf96653e-75ed-433e-a50b-494260c8f1b2)

Now it looks like this:
![image](https://github.com/user-attachments/assets/2e80d7e3-0eba-47d1-9544-4175fec890dc)

**Comment by akshayka** [comment]
cc @mscolnick

**Reply by mscolnick** [reply]
I can look into adding this back

**Reply by pyinstallerbaggins** [reply]
@mscolnick hello kind sir! any update on the original date picker. In the update, is it possible to choose date picker style?

thanks!

**Reply by mscolnick** [reply]
hi @pyinstallerbaggins - i havent had time to take a look at this. and some other work has taken precedence. we would be open to a contribution if you are up for it.
